hl7_receiving.py

```
@@ -49,20 +52,80 @@ def wrap_with_mllp(message: str) -> bytes:
     """
-    os.makedirs(response_dir, exist_ok=True)
+
+    if data.startswith(MLLP_START) and data.endswith(MLLP_END):
+        raw = data[1:-2]
+        message = raw.decode().replace('\n', '\r')
+
     timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+
+    try:
+        m = parse_message(message, find_groups=False)
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)